scripts/db_migrate.sh

```diff
@@ -35,7 +42,7 @@ case $DB_MODE in
         fi
 
         echo -n "Copying...! $NEWLINE"
-        mysqldump "$DB_EXPORT_FLAGS" --user="${DB_USER}" --password="${DB_PASS}" --host="${DB_HOST}" --port="${DB_PORT}" "${DB_NAME}" | mysql "$DB_IMPORT_FLAGS" --user="${DB_USER}" --password="${DB_PASS}" --host="${DB_HOST}" --port="${DB_TO_PORT}" "${DB_TO_NAME}"
+        docker exec -i "$CONTAINER_NAME_FROM" mysqldump $DB_EXPORT_FLAGS --user="${DB_USER}" --password="${DB_PASS}" --port="${DB_PORT}" --databases "${DB_NAME}" | docker exec -i "$CONTAINER_NAME_TO" mysql --user="${DB_USER}" --password="${DB_PASS}" --port="${DB_TO_PORT}" "${DB_TO_NAME}"
```

Comment:
I guess we either remove `DB_IMPORT_FLAGS` or add it back to command since its presently unused.

Reply:
Got it removed.